### PR TITLE
merge cucumbers [SCI-4798]

### PR DESCRIPTION
**Pull request by urbanrotnik**
Jira ticket: [SCI-4798](https://biosistemika.atlassian.net/browse/SCI-4798)

### CODE REVIEW

File: features/step_definitions/projects_page_steps.rb [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+Given('I click to {string} of a Options modal window') do |link|
+  page.find('.panel-project .dropdown-menu', text: 'Options').find('a', text: link).click
+end
+
+Then('I click {string} icon on {string} project card') do |icon, project|
+  page.find('.panel-project', text: project).find(".fa-#{icon}").click
+end
+
+Then('I select user {string} in user dropdown of user manage modal for project {string}') do |user, project|
+  within('.modal-content', text: "Manage users for #{project}") do
```
Comment (aignatov-bio) [inline features/step_definitions/projects_page_steps.rb:57]: i18n

File: features/step_definitions/projects_page_steps.rb [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    find('.btn[data-id="user_project_user_id"]').click
+    find('.dropdown-menu.open a', text: user).click
+  end
+end
+
+Then('I select role {string} in role dropdown of user manage modal for project {string}') do |role, project|
+  within('.modal-content', text: "Manage users for #{project}") do
```
Comment (aignatov-bio) [inline features/step_definitions/projects_page_steps.rb:64]: i18n

File: features/step_definitions/projects_page_steps.rb [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    find('.btn[title="Select Role"]').click
+    find('.dropdown-menu a', text: role).click
+  end
+end
+
+Then('I change role {string} in role dropdown for user {string} of user manage modal for project {string}') do |role, user, project|
+  within('.modal-content', text: "Manage users for #{project}") do
```
Comment (aignatov-bio) [inline features/step_definitions/projects_page_steps.rb:71]: i18n

File: app/views/projects/index.html.erb [hunk 4]
```diff
@@ -22,7 +22,7 @@
             </div>
             <div class="modal-footer">
               <button type="button" class="btn btn-secondary" data-dismiss="modal"><%=t "general.cancel" %></button>
-              <%= f.submit t("projects.index.modal_new_project.create"), class: "btn btn-primary" %>
+              <%= f.button t("projects.index.modal_new_project.create"), class: "btn btn-primary" %>
```
[the remark this answers is not in the document]
Comment (urbanrotnik) [reply app/views/projects/index.html.erb:25]: Yes

File: features/step_definitions/projects_page_steps.rb (hunk 2 above)
Comment (urbanrotnik) [reply features/step_definitions/projects_page_steps.rb:64]: Scenarios should be improved with common actions, then this file can be removed. No improvements for now.

File: features/step_definitions/projects_page_steps.rb (hunk 1 above)
Comment (urbanrotnik) [reply features/step_definitions/projects_page_steps.rb:57]: Scenarios should be improved with common actions, then this file can be removed. No improvements for now.

File: features/step_definitions/projects_page_steps.rb [hunk 5]
```diff
@@ -0,0 +1,163 @@
+# frozen_string_literal: true
+
+Then('I should see {string} public project card in {string} team page') do |project, team|
+  expect(page).to have_selector '.nav-name', text: team
+  expect(page).to have_selector '.panel-project a', text: project
+  expect(page).to have_selector '.panel-project span', text: 'All team members'
```
[the remark this answers is not in the document]
Comment (urbanrotnik) [reply features/step_definitions/projects_page_steps.rb:6]: Scenarios should be improved with common actions, then this file can be removed. No improvements for now.

File: features/step_definitions/projects_page_steps.rb (hunk 3 above)
Comment (urbanrotnik) [reply features/step_definitions/projects_page_steps.rb:71]: Scenarios should be improved with common actions, then this file can be removed. No improvements for now.

File: features/step_definitions/project_steps.rb [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+Then('I am on Reports page of {string} team of current user') do |team|
+  check_active_team(team)
+  expect(page).to have_selector 'a', text: 'New report'
+end
+
+Then('I am on Activities page of {string} team of current user') do |team|
+  check_active_team(team)
+  expect(page).to have_selector '.ga-title', text: 'Global activities'
+end
```
[the remark this answers is not in the document]
Comment (urbanrotnik) [reply features/step_definitions/project_steps.rb:38]: Scenarios should be improved with common actions, then this file can be removed. No improvements for now.

**Pull request merged**